app/controllers/admin/terms_of_service_files_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Admin
+  class TermsOfServiceFilesController < Spree::Admin::BaseController
+    def show
+      @current_file = TermsOfServiceFile.current
+      @new_file = TermsOfServiceFile.new
+    end
+
+    def new
+      show
+      render :show
+    end
+
+    def create
+      TermsOfServiceFile.create!(file_params)
+      redirect_to main_app.admin_terms_of_service_files_path
+    rescue ActionController::ParameterMissing
+      flash[:error] = t(".select_file")
```

Comment:
CodeCov is complaining these lines in the rescue block don't have any coverage. Maybe a quick test for the "unhappy" path here would be good? It would also presumably clarify the case where we expect `ActionController::ParameterMissing` to be thrown during file creation.